scarf/datastore.py

```diff
@@ -6,6 +6,7 @@
 """
 
 import os
+from re import L
```

Comment:
Not sure what `L` is here. Can you use `as` to rename it to something that will potentially not conflict with other variables in namespace.

Reply:
Sorry about that. Will remove it since it's not used anywhere.